### DISCUSSION TITLE
关于文件存储格式

### DISCUSSION THREAD

**Discussion by abj1210** [liusy58/NJU_AdvancedProgramming_SP22#80] [Q&A] [answered]
想问一下在存储数据的时候一定要按照ppt给出的格式进行存储吗（比如用户id一定存储为U001而不能为001或者1）？

**Comment by binbin00** [comment, marked as answer]
必须按照给定的要求存储。

**Reply by abj1210** [reply]
那可不可以在文件中另加其他信息呢

**Reply by binbin00** [reply]
> 那可不可以在文件中另加其他信息呢

你想加什么信息？和拓展功能相关？

**Reply by abj1210** [reply]
是关于基本功能的，加入后方便判断

**Reply by abj1210** [reply]
比如存储用户是否为管理员

**Reply by binbin00** [reply]
> 比如存储用户是否为管理员

用户不会是管理员啊，管理员只有一位，可以直接写死在代码中，没必要存。

**Reply by abj1210** [reply]
那加入我在之后想要加入注册管理员的功能，能不能再一开始就把用户类型存储到文件里呢

**Reply by binbin00** [reply]
> 那加入我在之后想要加入注册管理员的功能，能不能再一开始就把用户类型存储到文件里呢

不提倡加这个功能，若真想加，可以新建一个类似user.txt的文件，没必要存一起。

**Reply by raindrops-0199** [reply]
请问助教，用户、商品信息、订单三个文件可不可以按二进制存储呢（SQL语句仍然按文本存储）

**Reply by RicoloveFeng** [reply]
> 请问助教，用户、商品信息、订单三个文件可不可以按二进制存储呢（SQL语句仍然按文本存储）

ppt有说明，全使用txt格式。
